miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/rest/RestExceptionHandler.java

```diff
@@ -88,9 +90,25 @@ else if (exception instanceof RestException) {
   @JsonInclude(JsonInclude.Include.NON_NULL)
   public static class RestError {
     
+    public static enum DataFormat {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this redundant "static" qualifier. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2786)